noir-projects/aztec-nr/aztec/src/utils/comparison.nr

```diff
@@ -0,0 +1,37 @@
+struct ComparatorEnum {
+    EQ: u8,
+    NEQ: u8,
+    LT: u8,
+    LTE: u8,
+    GT: u8,
+    GTE: u8,
+}
+
+global Comparator = ComparatorEnum {
+    EQ: 1,
+    NEQ: 2,
+    LT: 3,
+    LTE: 4,
+    GT: 5,
+    GTE: 6,
+};
+
+pub fn assert_comparison<let N: u32>(lhs: Field, operation: u8, rhs: Field, error_msg: str<N>) {
```

Comment:
Created this utility function as now it's needed in more places than just note_getter.